.github/workflows/console-preview.yml

```diff
@@ -0,0 +1,71 @@
+name: Console Preview
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - closed
+    paths-ignore:
+      - "docs/**"
+      - "logo/**"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.head.ref || github.ref }}
+  cancel-in-progress: true
+
+env:
+  FLY_API_TOKEN: ${{ secrets.FLY_API_TOKEN }}
+  FLY_VERSION: "0.1.80"
+  APP_NAME: wing-console-pr-${{github.event.number}}
+
+jobs:
+  create:
+    name: "Create"
+    runs-on: ubuntu-latest
+    if: ${{ github.event.pull_request.head.repo.full_name == github.repository && github.event.action != 'closed' }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Install flyctl
+        uses: superfly/flyctl-actions/setup-flyctl@master
+        with:
+          version: ${{env.FLY_VERSION}}
+      - name: Deploy to Fly.io
```

Comment:
Is there a way to better integrate this with our build system and workflow? Seems like the entire build happens in a new container, but really we could be using the build artifact that we are already creating as part of PRs.

Worst case scenario, the build that's happening in the container should at least make use of the turbo remote caching